Sources/MongoSwift/Operations/CountOperation.swift

```diff
@@ -73,7 +73,7 @@ internal struct CountOperation<T: Codable>: Operation {
         let count = mongoc_collection_count_with_opts(
             self.collection._collection, MONGOC_QUERY_NONE, self.filter._bson, 0, 0, opts?._bson, rp, &error)
 
-        if count == -1 { throw parseMongocError(error) }
+        if count == -1 { throw getMongoError(error: error) }
```

Comment:
while you're here, using `guard` seems more idiomatic than an `if`

Reply:
fixed